core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml

```diff
@@ -42,6 +47,13 @@
       vars:
         file_name: "{{ vault_file_name }}"
 
+    - name: Download Vault Agent Helm Chart tar
+      include_role:
+        name: download
+        tasks_from: download_file
+      vars:
+        file_name: "v0.4.0.tar.gz"
```

Comment:
This seems to be a hardcoded value... :( Can we do something about that?

Reply:
Yes, and we will. :D